e2e/cli-e2e/tests/help.e2e.test.ts

```diff
@@ -14,12 +18,13 @@ describe('CLI help', () => {
 
   it('should produce the same output to stdout for both help argument and help command', async () => {
     const helpArgResult = await executeProcess({
-      command: 'code-pushup',
-      args: ['help'],
+      command: 'npx',
+      args: ['@code-pushup/cli', 'help'],
     });
```

Comment:
You don't set `cwd` here, I'm guessing that's a mistake?

Reply:
Yes!! Thx nice catch!